tests/drop_counters/fanout/mellanox/mellanox_fanout.py

```diff
@@ -0,0 +1,72 @@
+import pytest
+from ..fanout_base import BaseFanoutHandler
+
+MAX_OPENFLOW_RULE_ID = 65535
+
+class FanoutHandler(BaseFanoutHandler):
+    def __init__(self, testbed_devices):
+        self.initialized = False
+        self.rule_id = MAX_OPENFLOW_RULE_ID
+        self.run_ansible_cli_cmd = "cd {ansible_path}; ansible-playbook {playbook} -i lab -l {fanout_host} --extra-vars \"{extra_vars}\" -vvvvv"
+        # Ansible localhost fixture which calls ansible playbook on the local host
```

Comment:
Minor suggestion, I would just make the paths and filenames global variables like `MAX_OPENFLOW_RULE_ID` rather than including all these comments. They're helpful, but I think descriptive variables names would be cleaner. :)

Reply:
Fixed.